docs/reference/services.rst

```diff
@@ -0,0 +1,63 @@
+=================
+MicroOVN Services
+=================
+
+This document provides brief explanation of services provided by MicroOVN. They
+are generally not meant for direct user interaction and should be used only
+if necessary. List and status of these services can be displayed by running:
+
```

Comment:
This page presents a list of all MicroOVN services. The descriptions presented here is for reference only - the user is not expected to interact directly with these services.

The status of all services is displayed by running: